spec/models/event_spec.rb

```diff
@@ -365,4 +365,21 @@ def create_week_based_event(time)
 
     gevent.destroy
   end
+
+  describe 'duration wandering away bug #360' do
+    before do
+      @event = FactoryGirl.create(:simple, schedule_yaml: "---\n:start_date: 2011-08-07 13:13:00.000000000 +02:00\n:duration: 10800\n:rrules: []\n:exrules: []\n:rtimes:\n- 2011-08-08 19:00:00.000000000 +02:00\n- 2011-11-15 19:00:00.000000000 +01:00\n- 2012-02-06 19:00:00.000000000 +01:00\n:extimes: []\n")
+      @current_zone = Time.zone
+      Time.zone = ActiveSupport::TimeZone["Vancoover"]
```

Comment:
I'm not sure if your deduction that the spec is bogus is correct. It is also possible that this line is superfluous :smile: I will try if without this line the spec is failing without the changes in the other file.